Writerside/topics/Configuration.md

```diff
@@ -0,0 +1,160 @@
+settings.  
+It's only possible to add a new clocking according the amount of days that are set in the apps settings.  
+If mistakes were made an employee has to ask an admin to [change registered hours](#change-employee-hours) of
+days further in the past.
+
+![Register-second-time](ESS_registration_hours_second_time_without_client.jpg)
+
+![Adept-registered-time-step1](ESS_registration_hours_modify_previous_entry_without_client.jpg)
+
+### Register time for a client and a project
+
+![Register-time](ESS_registration_hours_first_time.jpg)
+
+When selecting a specific day on the [calendar](#calendar) you will be guided to the page where you'll be able to
+register worked time per client and project for that specific day.  On the page the expected work hours for that
+day for the user are shown, the amount depends on the statute of the user and the selected day. When a registration is
+made the total registered time will also be shown and if a break has been registered. When adding worked hours the start time
+for the new time registration will be set to the end time of the previous registration. When adding worked hours an
+employee has to choose a client out of a list of active clients and a project out of a list of active projects.  
+It is possible to edit previously registered worked time only for the amount of days that are set in the apps
+settings.  
+It's only possible to add a new clocking according the amount of days that are set in the apps settings.  
+If mistakes were made an employee has to ask an admin to [change registered hours](#change-employee-hours) of
+days further in the past.
+
+![Register-second-time](ESS_registration_hours_second_time.jpg)
+
+![Adept-registered-time-step1](ESS_registration_hours_modify_previous_entry.jpg)
+
+### Absence
+
+![Register-absence](ESS_registration_absence.jpg)
+
+Absences can only be added for the current, for future days or for the amount of previous days that the admin has set in
+the apps settings.  
+An employee will not be able to change absences of days further in the past. They will need to ask an admin
+to [change the absences](#change-employee-hours)
+
+You can choose between 4 different absences:
+
+- Legal holiday
+- Absence
+- Bank holiday
+- Other
+
+When selecting other, a text field will become available to explain the reason of absence.
+
+### Absences longer then one day
+
+After choosing an absence a datepicker will become available. The user can optionally select a future end date for the
+absence. When committed the absences are only shown on working days, days when work hours are expected.
+
+When choosing a day in the past a message pops up. The user can then choose a new and valid date or add an absence only
+for the selected day
+
+### Prickings
+
+Prickings can be accessed through the navigation bar on the left of the screen. The button will always display a clock symbol, but its text may vary.
+
+![Alt text](/Resources/pricking_timer_no_existing.png)  
```

Comment:
The images in this md start at path "Resources". I don't think that's exists here